### DISCUSSION TITLE
Rename column in board view crashes project

### DISCUSSION THREAD

**Discussion by tdubs42** [community/community#16105] [Projects and Issues]
When you click the ... and then "rename" it says "This project failed to load"

**Comment by jvalkeal** [comment]
Looks like it depends on length and how many characters you remove/add
![gh-projects-rename-error](https://user-images.githubusercontent.com/50398/170710029-9ea05320-2ea7-4670-825e-59f1f11c98bd.gif)

**Comment by jvalkeal** [comment]
Ah magic number is 8 characters. If you create board as `12345678` it fails immediately with rename, less characters and it depends what you type.
